downloadable/playercards/Unofficial Return to Night of the Zealot  Investigator Expansion.json

```diff
@@ -0,0 +1,10048 @@
+                          "Value": 0,
+                          "Locked": false,
+                          "Grid": true,
+                          "Snap": true,
+                          "IgnoreFoW": false,
+                          "MeasureMovement": false,
+                          "DragSelectable": true,
+                          "Autoraise": true,
+                          "Sticky": true,
+                          "Tooltip": true,
+                          "GridProjection": false,
+                          "HideWhenFaceDown": true,
+                          "Hands": true,
+                          "CardID": 105400,
+                          "SidewaysCard": false,
+                          "CustomDeck": {
+                            "1054": {
+                              "FaceURL": "https://arkham-horror-rtnotz-investigator-expansion.s3.eu-north-1.amazonaws.com/af6ed499-5441-457c-9a42-6c0775f77e5d-Front-Bloodwire%20%283%29-resolution-300-compressionquality-1.00-bleed--1.0-Front.jpg?cachebust=1757792838351",
+                              "BackURL": "https://steamusercontent-a.akamaihd.net/ugc/2342503777940352139/A2D42E7E5C43D045D72CE5CFC907E4F886C8C690/",
+                              "NumWidth": 1,
+                              "NumHeight": 1,
+                              "BackIsHidden": true,
+                              "UniqueBack": false,
+                              "Type": 0
+                            }
+                          },
+                          "LuaScript": "",
+                          "LuaScriptState": "",
+                          "XmlUI": ""
+                        },
+                        {
+                          "GUID": "e0004d",
+                          "Name": "Card",
+                          "Transform": {
+                            "posX": 0.0,
+                            "posY": 0.0,
+                            "posZ": 0.0,
+                            "rotX": 0.0,
+                            "rotY": 270.0,
+                            "rotZ": 0.0,
+                            "scaleX": 1.0,
+                            "scaleY": 1.0,
+                            "scaleZ": 1.0
+                          },
+                          "Nickname": "Bribed Silence (1)",
+                          "Description": "",
+                          "GMNotes": "{\"TtsZoopGuid\":\"b384ff18-2009-4ddd-9784-bc8dddbcd1b1\",\"id\":\"b384ff18-2009-4ddd-9784-bc8dddbcd1b1\",\"type\":\"Event\",\"class\":\"Rogue\",\"traits\":\"Illicit. Tactic.\",\"level\":1,\"agilityIcons\":1,\"willpowerIcons\":1}",
+                          "AltLookAngle": {
+                            "x": 0.0,
+                            "y": 0.0,
+                            "z": 0.0
+                          },
+                          "Tags": [
+                            "PlayerCard"
+                          ],
+                          "LayoutGroupSortInd
```

Comment:
These two lines should be removed (see comment at the top of the file). And I'd add an empty line at the end of the file.